src/scripts/main.js

```diff
@@ -1,3 +1,336 @@
+    if (previousElement && previousElement !== parentElement) {
+      previousElement.classList.remove('active');
+    }
+
+    parentElement.classList.add('active');
+    previousElement = parentElement;
+  }
+});
+// #endregion
+// #region form
+
+const form = document.createElement('form');
+const existingElement = document.querySelector('script');
+
+form.classList.add('new-employee-form');
+
+const labelsAndInputs = [
+  {
+    label: 'Name:',
+    type: 'text',
+    name: 'name',
+    data: 'name',
+  },
+  {
+    label: 'Position:',
+    type: 'text',
+    name: 'position',
+    data: 'position',
+  },
+  {
+    label: 'Age:',
+    type: 'number',
+    name: 'age',
+    data: 'age',
+  },
+  {
+    label: 'Salary:',
+    type: 'number',
+    name: 'salary',
+    data: 'salary',
+  },
+];
+
+// Додаємо лейбли та інпуты до форми
+labelsAndInputs.forEach((item) => {
+  const label = document.createElement('label');
+  const input = document.createElement('input');
+
+  label.textContent = item.label;
+  input.type = item.type;
+  input.name = item.name;
+  input.required = true;
+  input.setAttribute('data-qa', item.data);
+
+  label.appendChild(input);
+  form.appendChild(label);
+});
+
+// Створюємо новий елемент <select>
+const selectLabel = document.createElement('label');
+const select = document.createElement('select');
+const cities = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+
+select.name = 'city';
+selectLabel.textContent = 'Office:';
+select.required = true;
+
+cities.forEach((city) => {
+  const option = document.createElement('option');
+
+  option.value = city;
+  option.textContent = city;
+  select.appendChild(option);
+});
+
+const AgeLab = form.querySelector('label input[name="age"]').parentElement;
+
+selectLabel.appendChild(select);
+
+form.insertBefore(selectLabel, AgeLab);
+
+// Створюємо кнопку
+const submitButton = document.createElement('button');
+
+submitButton.type = 'submit';
+submitButton.textContent = 'Save to table';
+form.appendChild(submitButton);
+
+document.body.insertBefore(form, existingElement);
+
+// #endregion
+// #region check form
+document
+  .querySelector('.new-employee-form')
+  .addEventListener('submit', (even) => {
+    even.preventDefault();
+
+    // Збираємо дані з форми
+    let statusAnswers = true;
+    const formResult = even.target;
+    const names = formResult.querySelector('input[name="name"]').value;
+    const position = formResult.querySelector('input[name="position"]').value;
+    const city = formResult.querySelector('select[name="city"]').value;
+    const age = formResult.querySelector('input[name="age"]').value;
+    const salary = formResult.querySelector('input[name="salary"]').value;
+
+    // Валідація даних
+    if (names.length < 4) {
+      pushNotification('Error', 'Ups.\n ' + 'Your name too short', 'error');
+      statusAnswers = false;
+    }
+
+    if (age < 18 || age > 90) {
```

Comment:
The age validation condition is correct, but ensure that the age is parsed as a number. Currently, it might be treated as a string, which could lead to incorrect comparisons.